From f99359e33c7f7e44190317d01ac0e2fff6be7c0c Mon Sep 17 00:00:00 2001
From: Mateusz Koza <mateusz.koza@grinn-global.com>
Date: Thu, 12 Feb 2026 12:44:54 +0100
Subject: [PATCH] mtk_dp add support for direct display port interface

Fix provided by Mediatek to allow using DP interface via direct DP
connector.
---
 drivers/gpu/drm/mediatek/mtk_dp.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/mediatek/mtk_dp.c b/drivers/gpu/drm/mediatek/mtk_dp.c
index f384a2ce1d59..b590c05ffa85 100644
--- a/drivers/gpu/drm/mediatek/mtk_dp.c
+++ b/drivers/gpu/drm/mediatek/mtk_dp.c
@@ -2852,7 +2852,6 @@ static const struct drm_bridge_funcs mtk_dp_bridge_funcs = {
 	.mode_valid = mtk_dp_bridge_mode_valid,
 	.edid_read = mtk_dp_edid_read,
 	.detect = mtk_dp_bdg_detect,
-	.hpd_notify = mtk_dp_bridge_hpd_notify,
 };
 
 static void mtk_dp_debounce_timer(struct timer_list *t)
